backend/tournesol/views/unconnected_entities.py

```diff
@@ -27,42 +27,39 @@ class UnconnectedEntitiesView(
     serializer_class = EntityNoExtraFieldSerializer
     permission_classes = [IsAuthenticated]
 
-    _already_visited_node_ = set()
-    _all_connections_ = defaultdict(set)
-
-    def get_related_entities(self, entity_id):
-
-        self._already_visited_node_.add(entity_id)
-
-        related_entities = set()
-        if self._all_connections_.get(entity_id) is None:
-            return set()
+    def get_queryset(self):
+        # Get related entities from source entity
+        source_node = get_object_or_404(Entity, uid=self.kwargs.get("uid"))
+        comparison_pairs = list(
```

Comment:
Why not just `comparisons` ?

Reply:
I wanted to describe that this list contains pairs of entities id, but it may bring more confusion. Reverted.